app/assets/javascripts/edit_review.js

```diff
@@ -0,0 +1,60 @@
+$(".edit_button").click(function (event) {
+  event.preventDefault();
+  var href = this.href.split("/");
+  var review_id = href[href.length - 2];
+  if (this.text == "Edit Review") {
+    var rating_string = $('#review_rating' + review_id).text().split(" ");
```

Comment:
Identifier 'rating_string' is not in camel case.<br>Mixed double and single quotes.<br>Identifier 'review_id' is not in camel case.